### DISCUSSION TITLE
Language Translations

### DISCUSSION THREAD

**Discussion by yogeshpaliyal** [yogeshpaliyal/KeyPass#88] [Ideas]
Want this app in your local language? or want to contribute to translations?
https://hosted.weblate.org/projects/keypass/
Contribute on Weblate

**Comment by shuvashish76** [comment]
I've seen OSS projects use these sites for translation purpose
- [Crowdin](https://crowdin.com/)
- [Transifex](https://www.transifex.com/)
- [Weblate](https://weblate.org/en/)

I think Weblate is better as it's [opensource](https://github.com/WeblateOrg/weblate).

Features: https://weblate.org/en/features/
![Screenshot_2021-12-05-21-15-09](https://user-images.githubusercontent.com/48226312/144753433-4fc61e84-f12d-42d0-885e-1b9abd399061.jpeg)
https://weblate.org/en/hosting/

**Reply by yogeshpaliyal** [reply]
But ut is a paid service

**Reply by shuvashish76** [reply]
Weblate is completely free for libre-public projects. Setup your project there & request for Libre plan.

**Reply by yogeshpaliyal** [reply]
Cool, will request that

**Comment by yogeshpaliyal** [comment]
Registered on Weblate, anyone can contribute here

**Reply by shuvashish76** [reply]
[Here](https://hosted.weblate.org/engage/keypass/) 🎉
Request to add weblate link to app's about/help section.
